caffe2/core/db.cc

```diff
@@ -60,9 +60,9 @@ class MiniDBCursor : public Cursor {
     }
     // Actually read in the contents.
     CAFFE_ENFORCE_EQ(
-        fread(key_.data(), sizeof(char), key_len_, file_), key_len_);
+        int(fread(key_.data(), sizeof(char), key_len_, file_)), key_len_);
```

Comment:
This is a narrowing, unsigned->signed conversion. Can we just switch key_len to be size_t?